### PR TITLE
feat: allow to disable file uploads

### PR DESCRIPTION
**Pull request by thisisnithin**
## Motivation and Context

Config to disable file uploads.

## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: router/pkg/config/config.go [hunk 1]
```diff
@@ -124,11 +124,15 @@ type TrafficShapingRules struct {
 	Router RouterTrafficConfiguration `yaml:"router"`
 }
 
+type FileUploadsRules struct {
+	Enabled          bool        `yaml:"enabled" default:"true"`
```
Comment (StarpTech) [inline router/pkg/config/config.go:128]: Please provide env support for these configs `file_upload_enabled`

File: router/pkg/config/config.schema.json [hunk 2]
```diff
@@ -624,6 +624,30 @@
       "default": "/",
       "description": "The path of the GraphQL Playground. The GraphQL Playground is a web-based GraphQL IDE that allows you to interact with the GraphQL API. The default value is '/'."
     },
+    "file_uploads": {
```
Comment (StarpTech) [inline router/pkg/config/config.schema.json:627]: Use singular it is the feature "file_upload".

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: router/pkg/config/config.schema.json (hunk 2 above)
Comment (thisisnithin) [reply router/pkg/config/config.schema.json]: Done

File: router/pkg/config/config.go (hunk 1 above)
Comment (thisisnithin) [reply router/pkg/config/config.go]: Done

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**